dev.exs

```diff
@@ -28,6 +28,83 @@ Application.put_env(:phoenix_live_dashboard, DemoWeb.Endpoint,
   ]
 )
 
+defmodule DemoWeb.History do
+  use GenServer
+  alias Phoenix.LiveDashboard.TelemetryListener
+
+  @history_buffer_size 50
+
+  def data(metric) do
+    GenServer.call(__MODULE__, {:data, metric})
+  end
+
+  def start_link(args) do
+    GenServer.start_link(__MODULE__, args, name: __MODULE__)
+  end
+
+  def init(metrics) do
+    {:ok,
+     for metric <- metrics, reduce: %{} do
```

Comment:
extract this logic to a new handle_cast method I think to unblock init, since won't receive any calls until after we attach handlers, right?